### PR TITLE
DEVPROD-2285 Create Decomissioned Host tickets in DEVPROD project

### PR DESCRIPTION
**Pull request by khelif96**
DEVPROD-2285

### Description
Makes Decomissioned Host tickets go to the DEVPROD Jira project (Setting will be updated in admin settings)
Assigns to the runtimeEnvironments team
### Testing
https://jira.mongodb.org/browse/DEVPROD-2474

cc @brooookemiller

### CODE REVIEW

File: rest/route/host.go [hunk 1]
```diff
@@ -577,7 +577,7 @@ func (rh *hostProvisioningOptionsGetHandler) Run(ctx context.Context) gimlet.Res
 
 // //////////////////////////////////////////////////////////////////////
 //
-// GET /rest/v2/host/{host_id}/disable
+// POST /rest/v2/host/{host_id}/disable
```
Comment (bsamek) [inline rest/route/host.go:580]: nit, it's `hosts`, not `host`

File: units/deco_host_notify.go [hunk 2]
```diff
@@ -134,7 +136,10 @@ func (j *decoHostNotifyJob) Run(_ context.Context) {
 		Summary:     fmt.Sprintf("investigate automatically decommissioned host '%s'", j.Host.Id),
 		Type:        "Incident",
 		Description: strings.Join(descParts, "\n"),
-		Components:  []string{"Evergreen"},
+		Components:  []string{"Static Host Management"},
+		Fields: map[string]interface{}{
+			fieldAssignedTeams: []map[string]string{{"id": runtimeEnvironmentsId}}, // Assigned Teams: Runtime Environments
```
Comment (bsamek) [inline units/deco_host_notify.go:141]: I thought you had said "value", not "id", in Slack.

**Review by bsamek — APPROVED** [review]
lgtm with a nit and a question

File: units/deco_host_notify.go (hunk 2 above)
Comment (khelif96) [reply units/deco_host_notify.go:141]: @brooookemiller mentioned that its actually id since the value we are using for runtimeEnvironments is an id

Comment (ablack12) [reply units/deco_host_notify.go:141]: I had no idea we made tickets this way. It looks like we could set up a Splunk alert to create a ticket like this which I think would be way better than hardcoding it. I get the sense that this is a quick fix to get us over to the DEVPROD project but could you make a follow up ticket?

Comment (khelif96) [reply units/deco_host_notify.go:141]: Yes that was the intent behind this. Great idea! https://jira.mongodb.org/browse/DEVPROD-2486

**Pull request merged**